### PR TITLE
zh-CN: tutorials/keras/basic_text_classification.ipynb

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,713 @@
...
+    "colab_type": "text",
+    "id": "Eg62Pmz3o83v"
+   },
+   "source": [
+    "\n",
+    "此笔记本（notebook）使用评论文本将影评分为*积极（positive）*或*消极（nagetive）*两类。这是一个*二元（binary）*或者二分类问题，一种重要且应用广泛的机器学习问题。\n",
+    "\n",
+    "我们将使用来源于[网络电影数据库（Internet Movie Database）](https://www.imdb.com/)的 [IMDB 数据集（IMDB dataset）](https://www.tensorflow.org/api_docs/python/tf/keras/datasets/imdb)，其包含 50,000 条影评文本。从该数据集切割出的25,000条评论用作训练，另外 25,000 条用作测试。训练集与测试集是*平衡的（balanced）*，意味着它们包含相等数量的积极和消极评论。\n",
+    "\n",
+    "此笔记本（notebook）使用了 [tf.keras](https://www.tensorflow.org/guide/keras)，它是一个 Tensorflow 中用于构建和训练模型的高级API。有关使用 `tf.keras` 进行文本分类的更高级教程，请参阅 [MLCC文本分类指南（MLCC Text Classification Guide）](https://developers.google.com/machine-learning/guides/text-classification/)."
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "此笔记本（notebook）使用了 [tf.keras](https://www.tensorflow.org/guide/keras)，它是一个 Tensorflow 中用于构建和训练模型的高级API。有关使用 `tf.keras` 进行文本分类的更高级教程，请参阅 [MLCC文本分类指南（MLCC Text Classification Guide）](https://developers.google.com/machine-learning/guides/text-classification/)。"
```

Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]: Modified, please check.

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,713 @@
...
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/beta/tutorials/keras/basic_text_classification\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,713 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载 notebook</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载 notebook</a>\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,713 @@
...
+    "colab_type": "text",
+    "id": "hKY4XMc9o8iB"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/beta/tutorials/keras/basic_text_classification\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,713 @@
...
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Eg62Pmz3o83v"
+   },
+   "source": [
+    "\n",
+    "此笔记本（notebook）使用评论文本将影评分为*积极（positive）*或*消极（nagetive）*两类。这是一个*二元（binary）*或者二分类问题，一种重要且应用广泛的机器学习问题。\n",
+    "\n",
+    "我们将使用来源于[网络电影数据库（Internet Movie Database）](https://www.imdb.com/)的 [IMDB 数据集（IMDB dataset）](https://www.tensorflow.org/api_docs/python/tf/keras/datasets/imdb)，其包含 50,000 条影评文本。从该数据集切割出的25,000条评论用作训练，另外 25,000 条用作测试。训练集与测试集是*平衡的（balanced）*，意味着它们包含相等数量的积极和消极评论。\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "我们将使用来源于[网络电影数据库（Internet Movie Database）](https://www.imdb.com/)的 [IMDB 数据集（IMDB dataset）](https://tensorflow.google.cn/api_docs/python/tf/keras/datasets/imdb)，其包含 50,000 条影评文本。从该数据集切割出的25,000条评论用作训练，另外 25,000 条用作测试。训练集与测试集是*平衡的（balanced）*，意味着它们包含相等数量的积极和消极评论。\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 6]
```diff
@@ -0,0 +1,713 @@
...
+    "\n",
+    "此笔记本（notebook）使用了 [tf.keras](https://www.tensorflow.org/guide/keras)，它是一个 Tensorflow 中用于构建和训练模型的高级API。有关使用 `tf.keras` 进行文本分类的更高级教程，请参阅 [MLCC文本分类指南（MLCC Text Classification Guide）](https://developers.google.com/machine-learning/guides/text-classification/)。"
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "此笔记本（notebook）使用了 [tf.keras](https://tensorflow.google.cn/guide/keras)，它是一个 Tensorflow 中用于构建和训练模型的高级API。有关使用 `tf.keras` 进行文本分类的更高级教程，请参阅 [MLCC文本分类指南（MLCC Text Classification Guide）](https://developers.google.com/machine-learning/guides/text-classification/)。"
```

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 7]
```diff
@@ -0,0 +1,713 @@
...
+    "\n",
+    "影评——即整数数组必须在输入神经网络之前转换为张量。这种转换可以通过以下两种方式来完成：\n",
+    "\n",
+    "* 将数组转换为表示单词出现与否的由 0 和 1 组成的向量，类似于 one-hot 编码。例如，序列[3, 5]将转换为一个 10,000 维的向量，该向量除了索引为 3 和 5 的位置是 1 以外，其他都为 0。然后，将其作为网络的首层——一个可以处理浮点型向量数据的稠密层。不过，这种方法需要大量的内存，需要一个大小为 `num_words * num_reviews` 的矩阵。\n",
+    "\n",
+    "* 或者，我们可以填充数组来保证输入数据具有相同的长度，然后创建一个大小为 `max_length * num_reviews` 的整型张量。我们可以使用能够处理此形状数据的嵌入层作为网络中的第一层。\n",
+    "\n",
+    "在本教程中，我们将使用第二种方法。\n",
+    "\n",
+    "由于电影评论长度必须相同，我们将使用 [pad_sequences](https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) 函数来使长度标准化："
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
    "由于电影评论长度必须相同，我们将使用 [pad_sequences](https://tensorflow.google.cn/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) 函数来使长度标准化："
```

Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]: Sorry for these errors, I've made modifications, please check again.

File: site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb [hunk 8]
```diff
@@ -0,0 +1,713 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载 notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb]:
```suggestion
  },
  {
        "cell_type": "markdown",
        "metadata": {
           "id": "GEe3i16tQPjo",
           "colab_type": "text"
                },
        "source": [
           "Note: 我们的 TensorFlow 社区翻译了这些文档。因为社区翻译是尽力而为， 所以无法保证它们是最准确的，并且反映了最新的\n",
           "[官方英文文档](https://www.tensorflow.org/?hl=en)。如果您有改进此翻译的建议， 请提交 pull request 到\n",
           "[tensorflow/docs](https://github.com/tensorflow/docs) GitHub 仓库。要志愿地撰写或者审核译文，请加入\n",
           "[docs-zh-cn@tensorflow.org Google Group](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-zh-cn)。"
         ]
     },
```
It looks like there is nothing wrong with it, but I still have to ask you to join the disclaimer here. I am sorry for the trouble.

**Comment by Berrywrq** [comment]
Seems there is something wrong with my cla certification, how to solve it?

**Comment by Mr-Linus** [comment]
Don't worry, no problem.
LGTM

**Comment by Mr-Linus** [comment]
You promised my suggestion so google's CLA bot will detect that my Github email and CLAs email are inconsistent. Master Billy will help us solve. My fault.  😂 @lamberta

**Comment by lamberta** [comment]
Yes, CLAbot gets a bit fussy with different email addresses. I've verified the CLAs, no problem.
*Master Billy*—I'll put that on my business card 😄

**Comment by Berrywrq** [comment]
> OK, pull request should be good now.
> Needed to rebase from master since there was a merge conflict with your other PR for quickstart/advanced.
> Also, this notebook was missing a code cell that caused it to break. Added the training cell.
> Thanks

I‘m sorry for my mistakes, and thanks for your help!

**Pull request merged**